api/v1/server/embedded_spec.go

```diff
@@ -1262,6 +1266,31 @@ func init() {
         "type": "string"
       }
     },
+    "MonitorStatus": {
+      "description": "Status of the node monitor",
+      "properties": {
+        "Cpus": {
+          "description": "The number of CPUs to listen on for events.\n",
```

Comment:
The style in this Swagger spec is to avoid "The".
Also, remove the trailing newline.

    "description": "Number of CPUs to listen on for events.",